### DISCUSSION TITLE
Questions about "ng_carb_lifetime_cost_adjust(allt)"

### DISCUSSION THREAD

**Discussion by wanyingw193** [NREL/ReEDS-2.0#90] [Q&A: Data]
Hi,

I just wonder why ng_carb_lifetime_cost_adjust(allt) needs to adjust the capital cost for gas-cc to achieve net-zero? How to determine the ratio numbers in co2_cap_%capscenario%_ng_crf_penalty.csv. For example, co2_cap_2035_linear_ng_crf_penalty.csv has 100 in 2035, what is the reference of this number?

Thank you so much!

**Comment by wesleyjcole** [comment]
This ng_carb_lifetime_cost_adjust parameter is to help provide the model with foresight when solving sequentially.  The parameter increases the cost of natural gas units using the relative cost recovery factor, assuming that all costs would need to be recovered by the year the net zero target is reached.  This prevents the model from making new investments that become stranded assests.

**Reply by wanyingw193** [reply]
Thanks! Could I ask one more related question?

The updated cost_cap_mult for NGCC without CCS after applying ng_carb_lifetime_cost_adjust is:
cost_cap_fin_mult(i,r,t)$[gas(i)$(not ccs(i))] = cost_cap_fin_mult(i,r,t) * (((ng_carb_lifetime_cost_adjust(t) - 1) * .2) + 1) ; 

The updated cost_cap_mult for NGCC with and without CCS in Virgnia after applying ng_lifetime_cost_adjust is:
cost_cap_fin_mult(i,r,t)$[gas(i)$r_st(r,'VA')] = cost_cap_fin_mult(i,r,t) * ng_lifetime_cost_adjust(t) ;

Why the two equations are formed differently?

**Reply by vcelloho** [reply]
The reason that the first one includes a 20% multiplier is that's associated with the cost premium for building a renewable combustion turbine. The idea here is that we allow the model to build a plant that's gas powered but has the option to upgrade through a fuel conversion and operate under a 100% decarbonized power system. The VA policy could be updated to include a similar exemption, but predated the first equation.

**Comment by wanyingw193** [comment]
Hi, a few more questions about these two parameters.

(1) Does ng_crf_penalty.csv use the capacity recovery factor equation to determine the input numbers? If so, how is t in CRF determined ?

(2) Why are the input numbers for VA state and national net zero targets the same, even though the start and end years are different for the two cases?

(3) Is there a similar parameter that applies to coal-fired plants to allow them to recover all costs in the net-zero target year? If not, why would a coal-fired plant not need the similar parameter?

(4) RE-CT/RE-CC has the same OM cost and heat rate as NGCT/NGCC. The fuel price for natural gas is $2-$4/MMBtu and the H2 fuel price is $20/MMBtu. Does the fuel price for natural gas need to be adjusted as well?
